Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -1922,7 +1922,7 @@ String GraphicsContextGLANGLE::getProgramInfoLog(PlatformGLObject program)
     Vector<GLchar> info(length);
     GL_GetProgramInfoLog(program, length, &size, info.data());
 
-    GCGLsizei count;
+    GCGLsizei count = 0;
     PlatformGLObject shaders[2];
```

Comment:
Would be consistent to initialize the shaders[], too ?

Reply:
I only fixed what the static analyzer complained about!

But good point, the shaders follow the same unsafe paths, so it'd be better to make sure they're initialized.
Feel free to just fix it if you have a few minutes, otherwise I'll do it next week...

Reply:
-> https://bugs.webkit.org/show_bug.cgi?id=262028